.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: Bug report 🐛
 about: Report errors or unexpected behavior 🤔
-title: "[BUG] My bug report"
-labels: Issue-Bug
+title: "My bug report"
+labels: Issue-Discussion
```

Comment:
maybe question would be a better place to start

Reply:
Sorry, I don't understand the comment.

Reply:
I guess @TravisEz13 means the label `Issue-Question`